src/common/styles/componentsStyles.ts

```diff
@@ -50,7 +53,13 @@ export const Image = styled.img`
   object-fit: cover;
   height: 10rem;
   width: 10rem;
+  aspect-ratio: 1 / 1;
   border-radius: 10px;
+
+  @media (max-width: 710px) {
```

Comment:
it's better when all the media queries are consistent and coming from your theme